astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -19,7 +19,35 @@
 
 class SnowflakeOperatorAsync(SnowflakeOperator):
     """
-    Executes SQL code in a Snowflake database
+    - SnowflakeAsyncOperator uses the snowflake python connector ``execute_async`` method to submit a database command
+      for asynchronous execution.
+    - submit multiple queries in parallel without waiting for each query to complete.
+    - Accept List of query or  multiple query with ‘;’ semicolon separated string and params It loops through the
+      query and execute the query in sequence order and uses execute_async method to run the query
+    - Once the query is submitted it executes the query from one connection and get the query id from the
+      response and passed it trigger and close the connection. so that the worker slots are made free.
+    - The trigger gets the query id list as input and poll every few seconds to snowflake and check
+      for the query status based on the query id from different connection.
+
+    where can this operator fit in?
+         - Whether the single query or multiple query to execute the query in parallel and free up the worker resource.
```

Comment:
```suggestion
```

Reply:
Remove above as it is already covered in previous point

Reply:
Removed this line
`Whether the single query or multiple query to execute the query in parallel and free up the worker resource.`